lms/static/js/course_sharing/course_sharing_events.js

```diff
@@ -0,0 +1,29 @@
+/**
+* Module for emitting Course Sharing Events.
+*/
+(function(define) {
+    'use strict';
+
+    define(['jquery', 'logger'], function($, Logger) {
+        return function(courseId) {
+
+            $(".action-facebook[data-course-id='" + courseId + "']").on('click', function(){
+                // Emit an event telling that the Facebook share link was clicked.
+                Logger.log('edx.course.sharing.share_clicked', {
```

Comment:
Also, we need your opinion on it's naming convention and either it should be research or bi ? (from previous ticket i think it will be bi event?).